tests/test_exporter.py

```diff
@@ -35,6 +35,7 @@ def decoratee(param):
         exporter.force_flush()
 
     span = exporter.top_span.result(timeout=0.0)
+    assert span.attributes
     assert len(span.attributes.keys()) == 1
```

Comment:
```suggestion
    assert span.attributes
    assert span.attributes == {"param": 1}
```

Is this a more useful and stricter assertion for what is being tested here?

Reply:
Done